NineChronicles.Standalone.Executable/Program.cs

```diff
@@ -127,6 +135,17 @@ static void ConfigureSentryOptions(SentryOptions o)
                     o.InitializeSdk = false;
                 });
 #endif
+            AWSSink awsSink = null;
+            if (!(awsAccessKey is null) && !(awsSecretKey is null))
+            {
+                var credentials = new BasicAWSCredentials(awsAccessKey, awsSecretKey);
+                awsSink = new AWSSink(
+                    credentials,
+                    RegionEndpoint.APNortheast2,
```

Comment:
It would be better if we can set the region as configure too.

Reply:
Applied at 29b9b2f67984e3199c3efa05a6a7b95f50183c5a